### PR TITLE
Replace the 'SSL' broken link in ciphers doc with one pointing to security levels explained

### PR DESCRIPTION
**Pull request by EasySec**
SSL_CTX_set_security_level() seems not being referenced from elsewhere and an example using SECLEVEL is provided just above the 'SEE ALSO' section

### CODE REVIEW

File: doc/man1/ciphers.pod [hunk 1]
```diff
@@ -717,7 +717,7 @@ Set security level to 2 and display all ciphers consistent with level 2:
 
 =head1 SEE ALSO
 
-L<s_client(1)>, L<s_server(1)>, L<ssl(3)>
```
Comment (levitte) [inline doc/man1/ciphers.pod]: Section 3 is wrong, it's in section 7 (have a look in `doc/man7`) 

I think I would prefer if the section number was changed unless there's a strong reason to change the reference.

Comment (EasySec) [inline doc/man1/ciphers.pod]: You'r right. Sorry for missing that. May I still propose to add the link to 'security levels' which I found very usefull ?
PR changed.

Comment (EasySec) [inline doc/man1/ciphers.pod]: In such a case, is it better to modify the PR title ?

Comment (richsalz) [inline doc/man1/ciphers.pod]: i don't think ciphers is the place for the security-levels link.  it should be in man7/ssl.

Comment (EasySec) [inline doc/man1/ciphers.pod]: OK, I will update the PR.

**Comment by richsalz** [comment]
e330f55 on master; 66bf3bc on 1.1.0  thanks!

**Pull request closed without merging**